shell/platform/windows/keyboard_key_embedder_handler.cc

```diff
@@ -264,8 +264,17 @@ void KeyboardKeyEmbedderHandler::KeyboardHookImpl(
     pressingRecords_[physical_key] = eventual_logical_record;
   } else {
     auto record_iter = pressingRecords_.find(physical_key);
-    assert(record_iter != pressingRecords_.end());
-    pressingRecords_.erase(record_iter);
+    // Critical keys can be unset before this poitn if get_key_state_ finds they
+    // are not currently pressed. If the physical key of a keyup event is not in
+    // pressingRecords_, only assert that it is a critical key.
+    // This can occur when caps lock is double pressed with Windows Narrator
+    // running, as key events with wparam 0x14 (caps lock) but with scancode
+    // 0x00 are issued.
+    if (record_iter == pressingRecords_.end()) {
+      assert(critical_keys_.find(key) != critical_keys_.end());
```

Comment:
```suggestion
      assert(record_iter == pressingRecords_.end());
```
So that it's clearer that the `if` statement is just preventing run-time crashes.